spec/support/features/idv_step_helper.rb

```diff
@@ -43,8 +43,20 @@ def complete_idv_steps_before_phone_otp_verification_step(user = user_with_2fa)
   end
 
   def complete_idv_steps_with_phone_before_review_step(user = user_with_2fa)
-    complete_idv_steps_before_phone_step(user)
-    complete_phone_step(user)
+    if IdentityConfig.store.idv_api_enabled_steps.include?('password_confirm')
+      sign_in_and_2fa_user(user)
+      idv_session = Idv::Session.new(user_session: {}, current_user: user, service_provider: nil)
+      idv_session.applicant = Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE
+      idv_session.resolution_successful = true
+      idv_session.address_verification_mechanism = 'phone'
+      idv_session.user_phone_confirmation = true
+      idv_session.vendor_phone_confirmation = true
+      allow(Idv::Session).to receive(:new).and_return(idv_session)
```

Comment:
this looks similar but not quite to the usage below, what if we had a helper method?
```ruby
stub_idv_session(
  applicant: Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE,
  resolution_successful: true,
  address_verification_mechanism: 'phone'
  user_phone_confirmation: true,
  vendor_phone_confirmation: true,
)
```
and since Idv::Session just wraps a hash it might be easy, no need to add kwargs

Reply:
Dunno if it's quite what you had in mind for an implementation, but whipped something together in ef8873745.

Reply:
That's more elaborate than I was thinking but yes! It gets the job done